airbyte-integrations/connector-templates/source-python-http-api/source_{{snakeCase name}}/source.py.hbs

```diff
@@ -30,68 +30,112 @@ import requests
+    `class Employees({{properCase name}}Stream)` contains behavior to pull data for employees using v1/employees
+
+    If some streams implement incremental sync, it is typical to create another class
+    `class Incremental{{properCase name}}Stream(({{properCase name}}Stream), ABC)` then have concrete stream implementations extend it. An example
+    is provided below.
+
+    See the reference docs for the full list of configurable options.
+    """
+
     # TODO: Fill in the url base. Required.
-    url_base = 'fill-in-this-base'
+    url_base = 'https://example-api.com/v1/'
 
     def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
         """
-        TODO: Override this method to define a pagination strategy. Remove this method if not needed.
+        TODO: Override this method to define a pagination strategy. If you will not be using pagination, no action is required - just return None.
+
+        This method should return a Mapping (e.g: dict) containing whatever information required to make paginated requests. This dict is passed
+        to most other methods in this class to help you form headers, request bodies, query params, etc..
+
+        For example, if the API accepts a 'page' parameter to determine which page of the result to return, and a response from the API contains a
+        'page' number, then this method should probably return a dict {'page': response.json()['page'] + 1} to increment the page count by 1.
+        The request_params method should then read the input next_page_token and set the 'page' param to next_page_token['page'].
+
+        :param response: the most recent response from the API
+        :return If there is another page in the result, a mapping (e.g: dict) containing information needed to query the next page in the response.
+                If there are no more pages in the result, return None.
         """
         return None
 
     def request_params(self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, any] = None, next_page_token: Mapping[str, Any] = None) -> MutableMapping[str, Any]:
         """
-        TODO: Override this method to define any query parameters to be set. Remove this method if not needed. Usually contains common params e.g. pagination size etc.
+        TODO: Override this method to define any query parameters to be set. Remove this method if you don't need to define request params.
+        Usually contains common params e.g. pagination size etc.
         """
         return {}
 
 
-class MultipleShotPotatoGun({{properCase name}}Stream):
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+    """
+    TODO: Override this method to define how a response is parsed.
+    :return an iterable containing each record in the response
+    """
+        yield {}
+
+
+class Customers({{properCase name}}Stream):
```

Comment:
Changing the names seem fine to me. I had longer name examples to show how longer names are translated in the json. We can leave it as such if you think it's not necessary.

Reply:
we can also change to be something like `CustomerEvents`